tests/bud.bats

```diff
@@ -1797,3 +1797,10 @@ load helpers
   buildah rm --all
   buildah rmi --all --force
 }
+
+@test "bud using gitrepo and branch" {
+  target=gittarget
+  run_buildah bud --signature-policy ${TESTSDIR}/policy.json --layers -t ${target} git://github.com/containers/skopeo 
+
+  run_buildah bud --signature-policy ${TESTSDIR}/policy.json --layers -t ${target} git://github.com/containers/skopeo#master 
```

Comment:
I'm leery about adding these tests.  They're probably going to be close to timing out as I couldn't find a smaller repo to test against (happy to take suggestions), and if they complete, will add 10+ minutes to the test runs.